sdk/nodejs/admissionregistration/v1/MutatingWebhookConfiguration.ts

```diff
@@ -94,6 +94,13 @@ import { getVersion } from "../../version";
               opts.version = getVersion();
           }
 
-          super(MutatingWebhookConfiguration.__pulumiType, name, props, opts);
+          const _opts = pulumi.mergeOptions(opts, {
+              aliases: [
+                  { parent: opts.parent, type: "kubernetes:admissionregistration.k8s.io/v1:MutatingWebhookConfiguration", name: name },
```

Comment:
Isn't this an alias to itself?  I don't know for sure whether that causes any problem - but it seems like we shouldn't want/need to add it here?

Reply:
Yeah, I had done that previously for simplicity and it didn't seem to cause any problems. I should be able to remove that now.